internal/website/content/ref/runtimevar.md

```diff
@@ -13,7 +18,7 @@ Top-level package documentation: https://godoc.org/gocloud.dev/runtimevar
 * [GCP Runtime
   Configurator](https://godoc.org/gocloud.dev/runtimevar/runtimeconfigurator)
 * [blobvar](https://godoc.org/gocloud.dev/runtimevar/blobvar) - a blob-backed
-  implementation supported by any provider that has [blob support]({{< relref "/pages/blob.md#supported-providers">}})
+  implementation supported by any provider that has [blob support]({{< relref "blob.md#supported-providers">}})
```

Comment:
s/implementation/Implementation/?

Reply:
Part of a larger sentence, seems correct as-is?

Reply:
Sorry, missed that!